src/remix.ts

```diff
@@ -0,0 +1,94 @@
+import { Response } from "cross-fetch";
+import { z } from "zod";
+import {
+  InngestCommHandler,
+  serve as defaultServe,
+  ServeHandler,
+} from "./handlers/default";
+import { corsOrigin, fnIdParam, stepIdParam } from "./helpers/consts";
+
+/**
+ * app/inngest/index.server.ts
+ * app/routes/api/inngest.ts
+ */
+class RemixCommHandler extends InngestCommHandler {
+  protected override frameworkName = "remix";
+
+  public override createHandler() {
+    return async ({
+      request: req,
+    }: {
+      request: Request;
+    }): Promise<Response> => {
+      /**
+       * Specifically for CORS (browser->site requests), only allow PUT requests
+       * from the dashboard.
```

Comment:
FYI, we don't need this — it's our backend API that kicks off the handshake so we can leave all of our integrations CORS free!

Reply:
Ah aye - forgot to remove this following our changes